src/main/webapp/homepage_script.js

```diff
@@ -58,7 +58,7 @@ function addUIClickHandlers() {
             filterTasksBy(e.target.id);
         });
     }
-    // adds showTopScoresModal click event 
+    // adds showTopScoresModal click event
```

Comment:
It will be great to mention `closeTopScoresModal` click event here.

Reply:
Sounds good!